tasks/src/image-to-image/about.md

```diff
@@ -0,0 +1,36 @@
+## Use Cases
+
+### Style transfer
+
+One of the first and most popular applications, style transfer can convert a regular photography into a paining in the
+style of Claude Monet.
+
+### Image segementation
+
+Image segmentation represents is a pixel level classification - each pixel is assigned a numeric label, according to
+the object it belongs to.
+
+### Image inpainting
+
+Image inpaining is widely used during photography editing to remove unwanted objects, such as poles, wires or sensor
```

Comment:
I'm a bit inclined to have Image Inpainting as a sub-task in task variants, what do you think? 🙂 I feel like other use cases are general enough to be in task variants as well, except for Style Transfer.

Reply:
Considering the broadness of image to image translation, we can have many interpretations of what is a variant, and what really is the core application :). But I agree all are very broad applications.